collections/_drafts/agent-forwarding.md

```diff
@@ -0,0 +1,276 @@
+other GnuPG modules how to connect to the agent. However, correctly managing the
+startup and this environment variable is cumbersome, so a more straightforward
+method is required. Since GnuPG 2.0.16, the \--use-standard-socket option
+already allowed starting the agent on the fly; however, the environment variable
+was still needed.
+
+With GnuPG 2.1, the need for GPG_AGENT_INFO has been completely removed, and the
+variable is ignored.
+
+Instead, a fixed Unix domain socket named S.gpg\-agent in the GnuPG home
+directory (by default ~/.gnupg) is used. The agent is also started on-demand by
+all tools requiring services from the agent.
+
+If the option \--enable-ssh\-support is used, the auto-start mechanism does not
+work because _ssh_ does not know about this mechanism. Instead, the environment
+variable SSH_AUTH_SOCK must be set to the S.gpg-agent.ssh socket in the GnuPG
+home directory. Further, _gpg\_\_\-agent_ must be started by either using a
+GnuPG command that implicitly starts gpg\-agent or by using \`gpgconf --launch
+gpg\-agent\` to explicitly start it without first having to use a GnuPG command.
+
+gpg\-agent is a daemon to manage secret (private) keys independently from any
+protocol. It is a backend for gpg, gpgsm, and other utilities.
+
+GPG Agent Configuration
+
+There are a few configuration files needed for the operation of the agent. They
+may all be found in the current GnuPG home directory, which defaults
+to ~/.gnupg.
+
+\[\[windows\]\]
+
+However, there may be problems on Windows systems with Gpg4Win installed;
+**Gpg4Win may have changed this default GnuPG home directory location** to an
+AppData subdirectory (i.e., C:\\Users\\<username>\\AppData\\Roaming\\gnupg).
+This location, however, is not the location our GPG agent will be using, so to
+reaffirm our preference for the default location, we will set
+the GNUPGHOME environment variable to ~/.gnupg in the Git Bash startup script by
+running the following.
+
+echo 'export GNUPGHOME="~/.gnupg"' >> .bashrc
+
+\[\[endwindows\]\]
+
+Setting environment variables
+
+Add the following lines to your .bashrc or whatever initialization file is used
+for all shell invocations:
+
+GPG_TTY=$(tty)
+
+export GPG_TTY
+
+This variable may only be helpful if you use \`pinentry\-curses\` (the
+terminal-based pin entry program).
+
+\[\[windows\]\]
+
+On Windows systems, you should add the above lines to the ~/.bashrc file for use
+by Git Bash.
+
+\[\[endwindows\]\]
+
+GnuPG configuration
+
+It's important to note that GnuPG on the remote system still needs your public
+GPG keys to work correctly. So you have to ensure they are available on the
+remote system even if your secret keys are not.
+
+\[\[note\]\]
+
+If you use VSCode with the remote extension pack, you may skip this step and
+move on to the next section. This step of copying over your local public GPG
+keyring into the remote container gets done automatically by the extension.
+
+\[\[endnote\]\]
```

Comment:
```suggestion
:::note{.note}

If you use VSCode with the remote extension pack, you may skip this step and
move on to the next section. This step of copying over your local public GPG
keyring into the remote container gets done automatically by the extension.

:::
```

Refs: https://www.npmjs.com/package/remark-directive#example-styled-blocks

Reply:
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Markdown admonitions/badges, here we come. <a href="https://t.co/j07qwufKlH">https://t.co/j07qwufKlH</a></p>&mdash; FontVault (@FontVault) <a href="https://twitter.com/FontVault/status/1750744077337800742?ref_src=twsrc%5Etfw">January 26, 2024</a></blockquote>

Reply:
RE - ⬇️:

> re-reading https://github.com/OpenINF/GitHub-Markdown/issues/4 now and
> gearing up for its demolition, which we are running
> late, but kaboom happens in ~2hr 15min (10pm EST)
>
> who like me still feels weird about using the `aside` HTML tag for them?
>
> - http://html5doctor.com/understanding-aside/
> - https://developers.google.com/style/notices
> - https://docs.anaconda.com/restructuredtext/index.html#admonitions-notes
>
> maybe i will feel better about it after reading those things on them
>
> _Originally posted by @DerekNonGeneric in https://github.com/OpenINF/openinf.github.io/pull/1106#discussion_r1470413365_

GF - 🏛️:

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">opted for making old GF contiue her life in archive form<a href="https://t.co/J61RYALoyX">https://t.co/J61RYALoyX</a></p>&mdash; OpenINF (@OpenINF) <a href="https://twitter.com/OpenINF/status/1752173404289954250?ref_src=twsrc%5Etfw">January 30, 2024</a></blockquote>

Reply:
RE - ⬇️:

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">up next: we see about mirroring that yellow box as it should obv. be expressed in web tech for a custom admonitionn theme<a href="https://t.co/OB90Jk3bJL">https://t.co/OB90Jk3bJL</a></p>&mdash; OpenINF (@OpenINF) <a href="https://twitter.com/OpenINF/status/1752169541080084584?ref_src=twsrc%5Etfw">January 30, 2024</a></blockquote>

i see y'all on codepen in ~30min!

🔗 codepen.io/DerekNonGeneric

Reply:
> RE - ⬇️:
> 
> > up next: we see about mirroring that yellow box as it should obv. be expressed in web tech for a custom admonitionn themehttps://t.co/OB90Jk3bJL— OpenINF (@OpenINF) [January 30, 2024](https://twitter.com/OpenINF/status/1752169541080084584?ref_src=twsrc%5Etfw)
> 
> HW - 🏛️:
> 
> midnight eastern our classwork drop box self-destructs
> 
> i see y'all on codepen in ~30min!
> 
> 🔗 https://codepen.io/DerekNonGeneric



📖⚠️：https://codepen.io/DerekNonGeneric/pen/GReQKvr